boards/hip-badge/doc.txt

```diff
@@ -60,3 +51,5 @@ The event (and badge) were organized on short notice (6 Weeks), so there are som
 
  - [Badge Clinic](https://wiki.hip-berlin.de/index.php?title=Badge_Clinic)
  - [Design files](https://gitlab.com/tidklaas/hip-badge/)
+
+ */
```

Comment:
Do you really want to remove the Copyright notice? I think the only problem was that the closing `*/` was to at the wrong place in the file.

Reply:
I checked with other boards and they didn't have it and in general I'm for less boilerplate